### ISSUE TITLE
MKS TFT 35 90 degrees Screen Orientation

### ISSUE THREAD

**Issue by Cemileri** [makerbase-mks/MKS-TFT#438]
Hello everyone, I was wondering if there is a way to use MKS TFT 35 v 1.0  90 degrees vertical with all the buttons and text oriented in this way. Thanks in advance..

<img width="335" alt="Ekran Resmi 2020-08-06 14 27 24" src="https://user-images.githubusercontent.com/58373818/89526901-22d9a300-d7f1-11ea-93b1-16e605949da3.png">

**Comment by makerbase-mks** [comment]
Hi！ It is looks greatful, more users need to use it like this?

**Comment by Cemileri** [comment]
I think so..

**Comment by Cemileri** [comment]
> Hi！ It is looks greatful, more users need to use it like this?

Please add this functionality or show me a way to do it. Thx

**Comment by mahdieh-dev** [comment]
I have this problem too. Did you solve it?

**Comment by selvakumaran2016** [comment]
Yes, this will be useful for some machine makers, 90 deg rotation in both clockwise and anticlockwise direction

**Comment by mahdieh-dev** [comment]
@selvakumaran2016 Yes, exactly, my design needs this orientation. Did you find a way to implement it? I'm struggling for a week but no luck.

**Comment by selvakumaran2016** [comment]
@mshavandi Sorry, even I need that 90 deg orientation.

**Comment by tomasbaldi** [comment]
Yes, exactly, my design needs this orientation!!
